plugin.video.wizjatv/main.py

```diff
@@ -6,25 +6,28 @@
 PY2 = sys.version_info[0] == 2
 if PY2:
     from urlparse import parse_qs
+    from urllib import urlencode
 else:
-    from urllib.parse import parse_qs
+    from urllib.parse import parse_qs, urlencode
 
 from ptw.debug import log_exception, log, start_trace, stop_trace, TRACE_ALL
 import wizja
 
+# Addon Internal Modes:
+PLAY_LINK = 6
+
 obj = wizja.WizjaTvApi()
 s = requests.Session()
 
 def CATEGORIES():
     WizjaTV()
 
-def addDir(name, url, mode, thumb, fanart, opis, isFolder=True, total=1):
-    u=sys.argv[0]+'?url='+urllib.quote_plus(url)+'&mode='+str(mode)+'&name='+urllib.quote_plus(name)
-    ok = True
+def addDir(name, url, mode, thumb, fanart='', opis='', isFolder=True, total=1):
+    u = '{}?{}'.format(sys.argv[0], urlencode(dict(url=url, mode=mode, name=name)))
```

Comment:
To samo (tylko w druga stronę) co `parse_qs`. Jeśli standardowa biblioteka robi coś za nas to nie widzę sensu w robieniu tego na piechotę. Tutaj `urlencode()` tworzy query string (domyślnie spacjując plusami, czyli tak jak u nas).
Argumentem jest słownik. Czyli może być wcześniej konstruowany, można użyć `dict(key=value...)` albo `{key: value...}`.